.gitignore

```diff
@@ -107,26 +108,6 @@ venv.bak/
 # mypy
 .mypy_cache/
 
-# idea
+# editors
```

Comment:
A example of "git merge guessed wrong". Too much changed in the space of a few lines so `.vscode/` got moved away from its `# editors` comment. It makes no functional difference because .gitignore has a `\n` separated grammar, but it is interesting.